### DISCUSSION TITLE
installation stuck at `undo_telemetry.sh` script execution

### DISCUSSION THREAD

**Discussion by bnn1** [VSCodium/vscodium#1183] [General]
Hey guys. 

I've been waiting for like 30 minutes and it's still not over. 
Is it something to worry about? 
![image](https://user-images.githubusercontent.com/55663639/183230867-8bde0be3-27c5-4ef0-9fb7-e1b7692c4ea8.png)

upd. googled around and I think [that's fine](https://github.com/VSCodium/vscodium/issues/1165) =)

**Comment by daiyam** [comment]
Have you used the latest commit? On which system are you?
